helpers/focus.js

```diff
@@ -40,6 +40,25 @@ export function getFirstFocusableDescendant(node, includeHidden, predicate, incl
 	return null;
 }
 
+export function getFocusables(node, options) {
```

Comment:
I chose to accept an options argument instead. I'd actually like to update the other methods to clean up their API a bit, but that's a separate task.

This method doesn't drill into `svg` elements. This could be relaxed, but I was hoping to limit the DOM walking and I don't think we're ever expecting focusable elements inside of svgs.